index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="headwrap">
+    <header class="header">
+     <nav class="nav">
+         <i id="menu" class="material-icons">menu</i>
+     </nav>
+    </header>
+    <section class="headline">
+        <h1 class="name">Desmond
+            Wanjala</h1>
```

Comment:
- [ ] As said by the previous reviewer, kindly make sure you stick to the design as much as possible. Your page and the template have to look similar.
Kindly make sure you have these images on your design.
Your design :point_down: 
![Screenshot from 2022-10-28 08-58-28](https://user-images.githubusercontent.com/97100091/198524156-ee43eac8-f72b-4601-ac8b-583b00373014.png)

Figma template :point_down: 
![Screenshot from 2022-10-28 09-03-08](https://user-images.githubusercontent.com/97100091/198524529-ffe83364-9c7f-4df1-b52a-c407940f5884.png)

![Screenshot from 2022-10-28 09-04-44](https://user-images.githubusercontent.com/97100091/198524700-f1dee89e-ad9f-4263-99ba-63857a39664f.png)